docs/global.md

```diff
@@ -0,0 +1,70 @@
+# Global Options
+
+## Prefixes
+
+Terms in OBO and OWL are identified using <a href="https://en.wikipedia.org/wiki/Internationalized_resource_identifier" target="_blank">IRIs</a> (Internationalized Resource Identifiers), which generalize the familiar addresses for web pages. IRIs have many advantages, but one of their disadvantages is that they can be pretty long. So we have standard ways to abbreviate IRIs in a particular context by specifying **prefixes**. For example, Turtle files start with `@prefix` statements, SPARQL queries start with `PREFIX` statements, and JSON-LD data includes a `@context` with prefixes.
```

Comment:
Please use normal Markdown links throughout, not HTML links. We'll assume that people know how to use their browsers by now, and can open links in new windows/tabs if they want to.